www/src/pages/en/usage/env-variables.md

```diff
@@ -4,6 +4,8 @@ description: Getting started with create-t3-app
 layout: ../../../layouts/docs.astro
 ---
 
+## Typesafe Environment Variables
```

Comment:
I removed the immediate H2 cause I didn't deem it necessary, and often just repeated the H1. 

If you prefer this I'm totally cool just wanted to give my POV.

Reply:
I like this cause then we can get them in the right sidebar and we can directly link to those sections using ids

Reply:
Yes but the first h2 is the same as just linking to the page and often they basically say the same thing

Reply:
Hmm, I still like it if its there on the right lol.

Reply:
this is what i mean:
![CleanShot 2022-09-18 at 16 24 17](https://user-images.githubusercontent.com/51714798/190911968-851a2901-01e5-4b22-b1ba-8358bd22ad5c.png)

and for the links, this guy:

https://create-t3-nqwrgeebf-t3-oss.vercel.app/en/usage/env-variables

and this guy:

https://create-t3-nqwrgeebf-t3-oss.vercel.app/en/usage/env-variables#typesafe-environment-variables

is essentially the same..

> Hmm, I still like it if its there on the right lol.

we could change the TOC so that the first item is the H1 instead of just "Overview"

Reply:
> we could change the TOC so that the first item is the H1 instead of just "Overview"
Yeah maybe I would like that. We can do it in a different PR?

Reply:
sure :)